configs/fragments/bluetooth.fragment

```diff
@@ -0,0 +1,34 @@
+# Bluetooth Low Energy Support Fragment
+# Include this fragment in camera defconfigs to enable BLE functionality
+#
+# Usage in defconfig:
+# # FRAG: ... bluetooth
+
+# Enable Bluetooth utilities (requires BLE-capable WiFi driver)
+BR2_PACKAGE_THINGINO_BLUETOOTH=y
+
+# BlueZ 5 stack
+BR2_PACKAGE_BLUEZ5_TOOLS=y
+BR2_PACKAGE_BLUEZ5_UTILS=y
+BR2_PACKAGE_BLUEZ5_UTILS_TOOLS=y
+BR2_PACKAGE_BLUEZ5_UTILS_CLIENT=y
+
+# Enable deprecated tools (hcitool, hciconfig, hcidump, etc.)
+BR2_PACKAGE_BLUEZ5_UTILS_DEPRECATED=y
```

Comment:
This was not strictly required but it makes debugging a lot easier. You could disable it and still have all functionality working.